modules/tinymce/src/core/main/ts/ForceBlocks.ts

```diff
@@ -29,19 +29,19 @@ const isValidTarget = (blockElements, node) => {
   }
 };
 
-const hasBlockParent = (blockElements, root, node) => {
+const hasBlockParent = (blockElements: SchemaMap, root: Node, node: Node) => {
   return Arr.exists(Parents.parents(SugarElement.fromDom(node), SugarElement.fromDom(root)), (elm) => {
     return isBlockElement(blockElements, elm.dom);
   });
 };
 
 // const is
 
-const shouldRemoveTextNode = (blockElements, node) => {
+const shouldRemoveTextNode = (blockElements: SchemaMap, node: Node) => {
   if (NodeType.isText(node)) {
-    if (node.nodeValue.length === 0) {
+    if (node.data.length === 0) {
```

Comment:
Minor code change here and in a couple of places... `nodeValue` is nullable whereas `data` is not. They are the same thing for Text nodes though.